site/src/pages/api/blocks.api.ts

```diff
@@ -93,7 +93,7 @@ export const readBlocksFromDisk = (): BlockMetadata[] => {
   /* eslint-enable global-require */
 
   const registryInfo: BlockRegistryInfo[] = glob
-    .sync(`${process.cwd()}/../registry/**/*.json`)
+    .sync(`${process.cwd()}/../hub/**/*.json`)
```

Comment:
For consistency:
- [ ]  rename `BlockRegistryInfo` and `registryInfo` to `BuildConfig` and `buildConfig`

Reply:
Thanks. Will update